tests/webidl/post.js

```diff
@@ -1,4 +1,8 @@
 
+process.exit = (e) => {
+  throw e
```

Comment:
but I think the bigger question is, why is this needed in this test?

what worries me is that if we need such a specific hack here, then maybe our users may end up needing hacks of their own.

Reply:
Yeah, that's why I think this PR might not be what we want. try-catch in this way is only used in one test, but I think if we want this behavior it should be it's own test as well.

Reply:
Sorry, I don't follow - is a try-catch in this test a specific problem? which one?

Reply:
try-catch isn't the problem, it's just that it will exit on abort now, so it won't ever reach the other tests. try-catch was supposed to keep it from exiting.
But I will close this PR soon, I've found that the issue most likely lies in undefined symbol macro rather than abort itself.